src/experiment/trials/ibi.js

```diff
@@ -0,0 +1,127 @@
+import { videos } from "./introduction";
+import videoKeyboardResponse from '@jspsych/plugin-video-keyboard-response';
+
+const removeImages = () => {
+  const rightImg = document.getElementById('rdk-image-right');
+  const leftImg = document.getElementById('rdk-image-left');
+  if (rightImg !== null) {
+    rightImg.parentNode.removeChild(rightImg);
+  }
+  if (leftImg !== null) {
+    leftImg.parentNode.removeChild(leftImg);
+  }
+};
+
+const loadSpaceBarTapDiv = () => {
+  const video = document.getElementById(
+    'jspsych-video-keyboard-response-stimulus'
+  );
+
+  if (document.getElementById('space-bar-tap') === null) {
+    const tapDiv = document.createElement('div');
+    tapDiv.id = 'space-bar-tap';
+    tapDiv.onclick = () => {
+      buttonClicked = true;
+      pressKey(' ');
+    };
+    video.insertAdjacentElement('afterend', tapDiv);
+  }
+};
+
+// Inter block interval image
+export const IBI1 = {
```

Comment:
Can you consolidate these into a loop? Something like

const ibTrialData = [{stimulus: levelUpVideo1}, {stimulus: levelUpVideo12}]

const trialsMapped = ibTrialData.map(trial => {
  return (
    {
       type: videoKeyboardResponse,
  stimulus: trial.stimulus,
  prompt:
    '<p>Press the Spacebar when you are ready to proceed. Remember to sit at one arm distance from the screen.</p>',
  choices: [' '],
  response_allowed_while_playing: true,
  trial_duration: null,
  on_load: () => {
    removeImages();
    loadSpaceBarTapDiv();
  },
  width: 1238,
  height: 800,
    }
}

Then we can still access them the same throughout the experiment pulling them out
export const [IB1, IB2, etc] = trialsMapped